apps/social_feeds/lib/social_feeds/cache.ex

```diff
@@ -76,6 +76,11 @@ defmodule SocialFeeds.Cache do
 
   ## Server Callbacks
 
+  @doc """
+  Callback for starting the cache.
+  """
+  def init(state), do: {:ok, state}
```

Comment:
This callback was missing and it was causing an non-zero exit code when executing `mix project.check` in CI